### ISSUE TITLE
Upload to different repos

### ISSUE THREAD

**Issue by budius** [vanniktech/gradle-maven-publish-plugin#22]
I have some funny configurations on the project I work at, that makes me have to have multiple maven repo for upload (jcenter, and two different instances of private).

I've been working on making possibly to use as such on the build.gradle:

```
mavenRepo['repo1'] = {
	url = MAVEN_URL_1
	snapshotUrl = MAVEN_URL_SNAPSHOT_1
	username = MAVEN_USERNAME_1
	password = MAVEN_PASSWORD_1
}
mavenRepo['repo2'] = {
	url = MAVEN_URL_2
	snapshotUrl = MAVEN_URL_SNAPSHOT_2
	username = MAVEN_USERNAME_2
	password = MAVEN_PASSWORD_2
}
```

and this will create tasks `uploadRepo1` and `uploadRepo2`.

For the POM generation I've been using a fork of  Chris Banes original script, but as this repo is being maintained nicely (and I'm interested in this https://github.com/vanniktech/gradle-maven-publish-plugin/issues/21), I wonder if you accept Pull Request for that ?

**Comment by budius** [comment]
Well, I added the PR here https://github.com/vanniktech/gradle-maven-publish-plugin/pull/23
And I'm already using on our project using the generated .jar directly. Works super!

**Comment by vanniktech** [comment]
Thanks for your implementation and this can be closed now given that #23 is merged.
